### PR TITLE
Mdh issue 1797: more complex example for `<path>` element

### PR DESCRIPTION
**Pull request by martindholmes**
As detailed in issue #1797, I've added a new more complex example of the `<path>` element. A build of the PH chapter shows it directly following the example here:

https://hcmc.uvic.ca/~mholmes/tei/issue_1797/PH.html#index-egXML-d52e92324

### CODE REVIEW

File: P5/Source/Guidelines/en/PH-PrimarySources.xml [hunk 1]
```diff
@@ -380,7 +380,42 @@ used to define a polygon of any shape using this coordinate system:-->
             </surface></egXML>
          
          This is useful for linking an annotation or explanation to a specific line on
-         an object surface. Any number of coordinates can be included to specify lines which are not straight.
+         an object surface. Any number of coordinates can be included to specify lines 
+         which are not straight; this example shows the how the first of the famous
```
Comment (peterstadler) [inline P5/Source/Guidelines/en/PH-PrimarySources.xml]: there is a double "the" methinks …
```suggestion
         which are not straight; this example shows how the first of the famous 
```

**Comment by peterstadler** [comment]
is the zip file `sterne.zip` really necessary?

**Comment by martindholmes** [comment]
@peterstadler TCW 20 says:

> If you use an image in your Guidelines change, you will need to add it to the git repository in the P5/Images directory. If you have asked for and received permission from a rights-holder to use the image, include all of the relevant correspondence in a zip file named the same as the image file (so for an image fred.png, include permission documents in a file called fred.zip).

That's why the sterne.zip file is there. We should have them for all images which are not public property.

**Comment by martinascholger** [comment]
I like the example!

**Comment by peterstadler** [comment]
> > If you use an image in your Guidelines change, you will need to add it to the git repository in the P5/Images directory. If you have asked for and received permission from a rights-holder to use the image, include all of the relevant correspondence in a zip file named the same as the image file (so for an image fred.png, include permission documents in a file called fred.zip).
> 
> That's why the sterne.zip file is there. We should have them for all images which are not public property.

Ah, thanks for the explanation. I didn't know about that!

**Comment by peterstadler** [comment]
I think that can be merged, assigning @martinascholger to double check :)

**Comment by martinascholger** [comment]
Already approved :-) This can be merged.

**Pull request merged**